src/data/zwave_js.ts

```diff
@@ -29,10 +29,15 @@ export interface ZWaveJSNode {
 }
 
 export interface ZWaveJSNodeConfigParams {
+  [key: string]: ZWaveJSNodeConfigParam;
+}
+export interface ZWaveJSNodeConfigParam {
   property: number;
   value: any;
   configuration_value_type: string;
   metadata: ZWaveJSNodeConfigParamMetadata;
+  result?: string;
+  error?: string;
```

Comment:
We should not store this in `ZWaveJSNodeConfigParam`, as that is what we get from the backend. If we would refresh it, we lose our data. Create a new object.